### PR TITLE
[REFACTOR] auth-controllers

### PR DESCRIPTION
**Pull request by imdhruvgupta**

### CODE REVIEW

File: controllers/Auth/logout.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/**
+ * GET /logout
+ * Log out.
+ */
+
+exports.logout = function(req, res) {
+	req.session.destroy();
+	res.redirect('/dashboard/login');
+};
```
Comment (tarunsinghal92) [inline controllers/Auth/logout.js]: add a new line here

File: controllers/Auth/validateUser.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	users.getAllUsers({
+		'_id': new mongo.ObjectID(uid)
+	}, {}, function(users) {
+		if (users.length > 0) {
+			callback(users[0]);
+		} else {
+			callback(false);
+		}
+	});
+};
```
Comment (tarunsinghal92) [inline controllers/Auth/validateUser.js]: new line

**Comment by imdhruvgupta** [comment]
Hi @tarunsinghal92 , I have removed them.
Thanks

sugar

**Comment by tarunsinghal92** [comment]
@imdhruvgupta 

thanks for the changes!

could you please cherry-pick and make this pr to `dev` instead of `master`

**Comment by tarunsinghal92** [comment]
@imdhruvgupta 

can you please make the pr again?

1. move controllers inside the dashboard folder
2. delete any unnecessary code

**Comment by NikhilM98** [comment]
@imdhruvgupta `ini` in `getLogin.js` is not defined.

**Pull request merged**